src/Microsoft/SqlServer/Management/Smo/UserDefinedFunctionBase.cs

```diff
@@ -826,7 +826,9 @@ public override void Refresh()
                             "FunctionType",
                             "IsSchemaBound",
                             "IsNativelyCompiled",
-                            "IsEncrypted"};
+                            "IsEncrypted",
+                            "ReturnsNullOnNullInput",
```

Comment:
please sort the lists, and switch to nameof(propertyname) to match our updated standards.
Which reminds me, I need to add some more docs about these standards :-) We haven't copied all the internal content out here yet.

Reply:
I sorted & used `nameof` in most of the properties, but some are not direct properties of the `UserDefinedFunction` class in the library.

Reply:
@shueybubbles - is the fact that some properties are not generated (generate="false" in the cfg.xml file, i.e. impossible to use nameof() on) by design? It seems a bit odd that I'd have to resort to udf.Properties["DataTypeSchema"] to get it... but this is unrelated to the PR.

Reply:
yes some properties are "internal" and only set in the property bag but don't have C# wrappers.